docs/source/tutorials/token_classification.ipynb

```diff
@@ -116,11 +117,15 @@
    "metadata": {},
    "source": [
     "<details><summary>Below is the code used to read the `.npz` file. </summary> \n",
-    "    \n",
-    "    def read_npz(filepath): \n",
-    "        data = dict(np.load(filepath)) \n",
-    "        data = [data[str(i)] for i in range(len(data))] \n",
-    "        return data \n",
+    "\n",
+    "```ipython3\n",
+    "# Note: This pulldown content is for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
```

Comment:
Format for pulldown content copied from [multiannotator.ipynb](https://github.com/cleanlab/cleanlab/blob/f6d9fceddec4c34bf1a50c90bb040a36879d66ed/docs/source/tutorials/multiannotator.ipynb) tutorial.